### DISCUSSION TITLE
Setup won't proceed to Runtime installation on Ubuntu

### DISCUSSION THREAD

**Discussion by AdrianSingfield** [filips123/PWAsForFirefox#215] [Q&A]
I followed the instructions provided for Ubuntu on packagecloud.io but in the browser extension I'm still stuck on the Connector installation page. I've restarted Firefox several times and made sure the package was actually successfully installed. It still won't proceed to the next step. Anyone have an idea what's going on here and how I can get it to work?

Edit: I'm running a brand new install of Ubuntu MATE, the Firefox version is 104.0.2

**Comment by filips123** [comment]
Do you use Snap or Flatpak version of Firefox? If so, you have to switch to normal DEB package or manual install from the [Mozilla website](https://www.mozilla.org/firefox/all/#product-desktop-release), or switch to Firefox Snap beta as described [here](https://github.com/filips123/PWAsForFirefox/issues/76#issuecomment-1247300277).

I think you can check if you use Snap version if you go to Menu -> Help -> About Firefox.
